monasca-python/Dockerfile

```diff
@@ -3,5 +3,12 @@ ARG ALPINE_VERSION="3.6"
 
 FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
 
-COPY build.sh /
+COPY build.sh stack-fix.c /
 RUN chmod +x /build.sh
+
+RUN apk add --no-cache ca-certificates tini && \
+  apk add --no-cache --virtual build-dep musl-dev linux-headers git make g++ linux-headers && \
+  gcc -shared -fPIC /stack-fix.c -o /stack-fix.so && \
+  apk del build-dep
+
+ENTRYPOINT ["LD_PRELOAD=/stack-fix.so", "/sbin/tini", "--"]
```

Comment:
Couldn't we just ```entrypoint.sh``` like so:

```sh
#!/bin/sh
export LD_PRELOAD=/stack-fix.so
/sbin/tini -- "$@"
```

that's raw code, have no idea if it will work but if it does, will it be acceptable approach ?